app/locales/en/translations.js

```diff
@@ -1,5 +1,6 @@
 const preprints = `Preprints`;
 const brand = `OSF ${preprints}`;
+const permission_sentence1 = 'arXiv is a trademark of Cornell University, used under license.';
```

Comment:
Maybe `arxiv_trademark_license` for this and `arxiv_non_endorsement` for the other.